plugins/SnapshotPlugin/snapshot-helper.js

```diff
@@ -29,7 +29,7 @@ global.updateEnvironment = function() {
 
 	try {
 		// Mark the iLib localestorage cache as needing re-validation.
-		var ilib = require('@enact/i18n/ilib/lib/ilib');
+		var ilib = require('ilib/lib/ilib');
```

Comment:
does this need a fallback like the others?

Reply:
Does not since it's content-based (like apps), where we have alias fallbacks already in-place